audits/judging/judging/README.md

```diff
@@ -18,49 +18,57 @@ This guide aims to provide clarity for both Watsons & protocols on various categ
+   2. When `external-admin=restricted`, issues related to these external admins affecting a protocol (being audited) by updating **the external protocol parameters** is a **valid issue** (Example: Aave governance has the intention to improve the Aave protocol) as the bug can occur even when the external admin is well intended
 6. **Discord messages or DM** screenshots are not considered sources of truth while judging an issue/escalation especially if they are conflicting with the contest README.
 7. **Contract Scope:**
-   1. If a contract is in contest Scope, then all its parent contracts are included by default.
+   1. If a contract is in contest Scope, then all its parent contracts are included by default. 
    2. In case the vulnerability exists in a library and an in-scope contract uses it and is affected by this bug this is a valid issue.
    3. ﻿﻿If there is a vulnerability in a contract from the contest repository but is not included in the scope then issues related to it cannot be considered valid.
+8. **Opportunity Loss** is not considered a loss of funds by Sherlock. For example, loss of functionality is not considered a loss of protocol revenue, nevertheless issues involving opportunity loss may be valid issues (for example, due to a loss of core functionality).
+9. **Design decisions** are not valid issues. Even if the design is suboptimal, but doesn't imply any loss of funds, these issues are considered informational.
 
 ### IV. How to identify a high issue:
 
-1. Definite loss of funds without limiting external conditions.
-2. Breaks core contract functionality, rendering the protocol/contract useless (should not be easily replaced without loss of funds) and definitely causes significant loss of funds.
-3. Significant loss of funds/large profit for the attacker at a minimal cost.
+1. Definite loss of funds without (extensive) limitations of external conditions.
+2. Inflicts serious non-material losses (doesn't include contract simply not working).
 
 ### V. How to identify a medium issue:
 
-1. Causes a loss of funds but requires certain external conditions or specific states.
-2. Breaks core contract functionality, rendering the contract useless (should not be easily replaced without loss of funds) or leading to unknown potential exploits/loss of funds. \
-   Ex: Unable to remove malicious user/collateral from the contract.
-3. A material loss of funds, no/minimal profit for the attacker at a considerable cost
+1. Causes a loss of funds but requires certain external conditions or specific states, or a loss is highly constrained. The losses must exceed small, finite amount of funds, any amount relevant based on the precision (i.e. rounding to $10 ^ {-18}$ of the amount is fine), and 0.05% APY. Some concepts used in the README/codebase may override these parameters to more sensible ones depending on the context.
```

Comment:
> The losses must exceed small, finite amount of funds, any amount relevant based on the precision (i.e. rounding to $10 ^ {-18}$ of the amount is fine), and 0.05% APY. Some concepts used in the README/codebase may override these parameters to more sensible ones depending on the context.

This is kinda hard to comprehend for me, when reading it. What you want to say, is that the loss must exceed an "relevant" amount, according to the precision?